src/core_plugins/kibana/public/discover/index.html

```diff
@@ -151,17 +161,20 @@ <h1 tabindex="0" id="kui_local_breadcrumb" class="kuiLocalBreadcrumb">
                 data-shared-item
                 data-title="{{opts.savedSearch.lastSavedTitle}}"
                 data-description="{{opts.savedSearch.description}}"
+                minimum-visible-rows="minimumVisibleRows"
                 render-counter
                 on-add-column="addColumn"
                 on-change-sort-order="setSortOrder"
                 on-move-column="moveColumn"
                 on-remove-column="removeColumn"
               ></doc-table>
 
+              <a tabindex="0" id="discoverBottomMarker"></a>
+
               <div ng-if="rows.length == opts.sampleSize" class="discover-table-footer">
                 These are the first {{opts.sampleSize}} documents matching
                 your search, refine your search to see others.
-                <a ng-click="toTop()">Back to top.</a>
+                <a tabindex="0" ng-click="scrollToTop()">Back to top.</a>
```

Comment:
Since you're in here anyway, would you mind adding `kbnAccessibleClick` to this element? Currently if it has focus and you hit enter, nothing happens.